src/stcal/jump/twopoint_difference.py

```diff
@@ -291,7 +304,7 @@ def calc_med_first_diffs(first_diffs):
     row2, col2 = np.where(num_usable_groups == 2)  # locations of >= 4 usable group pixels
     if len(row2) > 0:
```

Comment:
I'm wondering if we should enclose the previous blocks of code that work with row4,col4 and row3,col3 in IF-blocks, just like we do here for row2,col2?  Based on one quick test, it appears that when row4=row3=0, the lines above don't do any damage, but it just seems safer to me to not even call lines 293-296 and 300-301 if row4=0 or row3=0.

Reply:
that sounds like a good thing to do, i'll go ahead and fix that